### ISSUE TITLE
WinError 2 The system cannot find the file specified

### ISSUE THREAD

**Issue by FrongTH** [ultralytics/yolov5#6884]
### Search before asking

- [x] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### Bug

![Screenshot 2022-03-07 170941](https://user-images.githubusercontent.com/89198355/157011013-71642342-341d-430d-9106-226598d7d9e2.png)
I start with anaconda prompt after that I create new env for test yolov5. 
conda install git
git clone https://github.com/ultralytics/yolov5
cd yolov5
pip install -r requirements.txt

and run code to test python detect.py
This error is appear....
How to fix that anyone can help.......... T-T

I find every way to fix that but I can't :
conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch from this https://pytorch.org/
Visual studio Build Tools 2022
pip install git+https://github.com/philferriere/cocoapi.git#subdirectory=PythonAPI

### Environment

python 3.8.2
anaconda

### Minimal Reproducible Example

absl-py                   1.0.0                    pypi_0    pypi
backcall                  0.2.0                      py_0    anaconda
blas                      1.0                         mkl
ca-certificates           2020.10.14                    0    anaconda
cachetools                5.0.0                    pypi_0    pypi
certifi                   2020.6.20                py38_0    anaconda
charset-normalizer        2.0.12                   pypi_0    pypi
colorama                  0.4.4                      py_0    anaconda
cudatoolkit               11.3.1               h59b6b97_2
cycler                    0.11.0                   pypi_0    pypi
cython                    0.29.21          py38ha925a31_0    anaconda
decorator                 4.4.2                      py_0    anaconda
fonttools                 4.29.1                   pypi_0    pypi
freetype                  2.10.4               hd328e21_0
git                       2.34.1               haa95532_0
google-auth               2.6.0                    pypi_0    pypi
google-auth-oauthlib      0.4.6                    pypi_0    pypi
grpcio                    1.44.0                   pypi_0    pypi
idna                      3.3                      pypi_0    pypi
importlib-metadata        4.11.2                   pypi_0    pypi
intel-openmp              2021.4.0          haa95532_3556
ipython                   7.18.1           py38h5ca1d4c_0    anaconda
ipython_genutils          0.2.0                    py38_0    anaconda
jedi                      0.18.1           py38haa95532_1
jpeg                      9d                   h2bbff1b_0
kiwisolver                1.3.2                    pypi_0    pypi
libpng                    1.6.37               h2a8f88b_0
libtiff                   4.2.0                hd0e1b90_0
libuv                     1.40.0               he774522_0
libwebp                   1.2.2                h2bbff1b_0
lz4-c                     1.9.3                h2bbff1b_1
markdown                  3.3.6                    pypi_0    pypi
matplotlib                3.5.1                    pypi_0    pypi
mkl                       2021.4.0           haa95532_640
mkl-service               2.4.0            py38h2bbff1b_0
mkl_fft                   1.3.1            py38h277e83a_0
mkl_random                1.2.2            py38hf11a4ad_0
numpy                     1.21.5           py38ha4e8547_0
numpy-base                1.21.5           py38hc2deb75_0
oauthlib                  3.2.0                    pypi_0    pypi
opencv-python             4.5.5.62                 pypi_0    pypi
openssl                   1.1.1h               he774522_0    anaconda
packaging                 21.3                     pypi_0    pypi
pandas                    1.4.1                    pypi_0    pypi
parso                     0.8.0                      py_0    anaconda
pickleshare               0.7.5                 py38_1000    anaconda
pillow                    9.0.1            py38hdc2b20a_0
pip                       21.2.2           py38haa95532_0
prompt-toolkit            3.0.8                      py_0    anaconda
protobuf                  3.19.4                   pypi_0    pypi
psutil                    5.9.0            py38h294d835_0    conda-forge
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pygments                  2.7.1                      py_0    anaconda
pyparsing                 3.0.7                    pypi_0    pypi
python                    3.8.2               he1778fa_13
python-dateutil           2.8.2                    pypi_0    pypi
python_abi                3.8                      2_cp38    conda-forge
pytorch                   1.10.2          py3.8_cuda11.3_cudnn8_0    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2021.3                   pypi_0    pypi
pyyaml                    6.0                      pypi_0    pypi
requests                  2.27.1                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
rsa                       4.8                      pypi_0    pypi
scipy                     1.8.0                    pypi_0    pypi
seaborn                   0.11.2                   pypi_0    pypi
setuptools                58.0.4           py38haa95532_0
six                       1.16.0             pyhd3eb1b0_1
sqlite                    3.37.2               h2bbff1b_0
tensorboard               2.8.0                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
thop                      0.0.31-2005241907          pypi_0    pypi
tk                        8.6.11               h2bbff1b_0
torchaudio                0.10.2               py38_cu113    pytorch
torchvision               0.11.3               py38_cu113    pytorch
tqdm                      4.63.0                   pypi_0    pypi
traitlets                 5.0.5                      py_0    anaconda
typing_extensions         3.10.0.2           pyh06a4308_0
urllib3                   1.26.8                   pypi_0    pypi
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
wcwidth                   0.2.5                      py_0    anaconda
werkzeug                  2.0.3                    pypi_0    pypi
wheel                     0.37.1             pyhd3eb1b0_0
wincertstore              0.2              py38haa95532_2
xz                        5.2.5                h62dcd97_0
zipp                      3.7.0                    pypi_0    pypi
zlib                      1.2.11               h8cc25b3_4
zstd                      1.4.9                h19a0ad4_0

### Additional

I have more Q? 
Why I run code in colab is not error but on cmd or anaconda prompt is error?

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@FrongTH `utils.notebook_init()` is just meant for Colab and Kaggle to display system info. You don't need to run it in your custom workflows.

**Comment by FrongTH** [comment]
So How can I work yolov5 for my cmd?
![noname](https://user-images.githubusercontent.com/89198355/157272276-c269f25d-a050-4fb5-b216-d898c0355740.png)
I installed git from here : https://git-scm.com/docs/git-describe
after that git clone https://github.com/ultralytics/yolov5
create new env
activate new env and pip install -r requirements.txt
>>> python detect.py --source 0 
Error like this How can I fix that

Thank you so much

**Comment by glenn-jocher** [comment]
@FrongTH it appears you may have environment problems. Please ensure you meet all dependency requirements if you are attempting to run YOLOv5 locally.  If in doubt, create a new virtual [Python 3.9](https://www.python.org/) environment, clone the latest repo ([code changes daily](https://github.com/ultralytics/yolov5/commits/master)), and `pip install` [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) again from scratch.

💡 ProTip! Try one of our verified environments below if you are having trouble with your local environment.


## Requirements

[**Python>=3.7.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed including [**PyTorch>=1.7**](https://pytorch.org/get-started/locally/). To get started:
```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install -r requirements.txt  # install
```

[Models](https://github.com/ultralytics/yolov5/tree/master/models) and [datasets](https://github.com/ultralytics/yolov5/tree/master/data) download automatically from the latest YOLOv5 [release](https://github.com/ultralytics/yolov5/releases) when first requested.

## Environments

YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Google Colab and Kaggle** notebooks with free GPU: <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>


## Status

![CI CPU testing](https://github.com/ultralytics/yolov5/workflows/CI%20CPU%20testing/badge.svg)

If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are currently passing. CI tests verify correct operation of YOLOv5 training ([train.py](https://github.com/ultralytics/yolov5/blob/master/train.py)), validation ([val.py](https://github.com/ultralytics/yolov5/blob/master/val.py)), inference ([detect.py](https://github.com/ultralytics/yolov5/blob/master/detect.py)) and export ([export.py](https://github.com/ultralytics/yolov5/blob/master/export.py)) on MacOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by FrongTH** [comment]
I did everything that your suggestion but still error
I changed the new directory to `d:` and using python 3.9.10
Created new venv and activate that
`git clone https://github.com/ultralytics/yolov5 `
I tried to `pip3 install torch==1.10.2+cu113 torchvision==0.11.3+cu113 torchaudio===0.10.2+cu113 -f https://download.pytorch.org/whl/cu113/torch_stable.html`
after that `pip install -r requirements.txt`
Run `python detect.py` and the same error

![1](https://user-images.githubusercontent.com/89198355/157376534-62e87346-b5fe-45d2-ba83-72a9f5517c31.jpg)
![2](https://user-images.githubusercontent.com/89198355/157376542-834d20f1-97fb-4110-ab8a-133a6dcfac80.jpg)
![3](https://user-images.githubusercontent.com/89198355/157376543-eb67bfbf-e12c-431c-9b83-98c0956c1753.jpg)
![4](https://user-images.githubusercontent.com/89198355/157376546-248541b1-c957-45cd-8140-56245aad0770.jpg)

**Comment by glenn-jocher** [comment]
@FrongTH try Docker
